product_packaging_container_deposit/tests/test_product_packaging_container_deposit.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from .common import Common
+
+
+class TestProductPackagingContainerDeposit(Common):
```

Comment:
pls add a TODO to test the mixins using fake model + odoo-test-helper to not depend on so and po tests from depending modules

Reply:
TODO removed as I added unit tests